planner:
  gripper: "hanyang"
  visualize: False
  show_finger_path: False
  dish_r_margin: 0.035 # unit: m
  swipe_a_margin: 5.0 # unit: deg
  swipe_speed: 0.07   # unit:m/s
  height: 0.015

gripper:
  hanyang:
    width: 0.10
    height: 0.343421
    finger_angle: 180 # hanyang
    z_angle: +13.1825

